src/ui/server/api/keystone/application/index.ts

```diff
@@ -276,6 +277,36 @@ const application = {
       }
     },
     declarations: declarations.update,
+    sectionReview: async (id: string, update: object) => {
+      try {
+        console.info('Updating application section review');
+
+        const variables = {
+          where: { id },
+          data: update,
+        };
+
+        const response = (await apollo('POST', updateApplicationSectionReviewMutation, variables)) as ApolloResponse;
+
+        if (response.errors) {
+          console.error('GraphQL error updating application section review ', response.errors);
+        }
+
+        if (response?.networkError?.result?.errors) {
+          console.error('GraphQL network error updating application section review ', response.networkError.result.errors);
+        }
+
+        if (response?.data?.updateSectionReview) {
+          return response.data.updateSectionReview;
```

Comment:
No need for it, this is the successful part - if all the data is returned  it'll return the data. Otherwise an error will be caught/logged in one of the other conditions.